static/js/map-around.js

```diff
@@ -1,6 +1,7 @@
 // POIS creation
 const worker = new Worker('/static/js/webworker-around.js')
 let pois = []
+let currentUserLocation
```

Comment:
Why do we need this here as `currentUserLocation` is only used in the `userLocalized` function?